### ISSUE TITLE
Jump to file under cursor not working

### ISSUE THREAD

**Issue by maiermic** [hrjakobsen/slext#56]
Jump to file under cursor does not work most of the time. I try to use it on an `\input{path/to/file.tex}` with the cursor between the curly braces. I use Chrome 71.0.3578.98 on Linux Mint. So I press `Alt + G`, but nothing happens even though `path/to/file.tex` should be opened.

**Comment by hrjakobsen** [comment]
Do you use relative paths such as `../path/to/file.tex`? Because they are currently not supported.

**Comment by maiermic** [comment]
No, my paths start always at the project root directory and are not relative to the file they are referred from.

**Comment by hrjakobsen** [comment]
It does not work with unix paths such as `/path/to/file`,`./path/to/file` or `../path/to/file`. This is something that I've wanted to implement for a while now. I will update the issue to reflect the problem.

**Comment by hrjakobsen** [comment]
Work has begun on this in b94f9e3473d. It will need some more testing, but I have made it possible to extract the file-tree structure from overleaf, so that we can traverse paths with `../` and `./`.
